src/pyglottolog/api.py

```diff
@@ -40,7 +65,12 @@ class Glottolog(API):
         'publication': config.Generic,
     }
 
-    def __init__(self, repos='.'):
+    def __init__(self, repos='.', cache=False):
```

Comment:
make `cache` a keyword-only-argument  (`, *, cache=`)?

Reply:
Good idea. Also good to remind me of what's possible with py3.